portfolio/src/main/webapp/script.js

```diff
@@ -65,7 +65,15 @@ function grow() {
 }
 
 function getGreeting() {
```

Comment:
`getGreetings`

Reply:
Will rename the variable `getComments` for later steps.